lib/from_openstudio/model.rb

```diff
@@ -119,5 +123,23 @@ def self.orphaned_shades_from_model(openstudio_model)
       result
     end
 
+    # Create HB Material from OpenStudio Materials
+    def self.materials_from_model(openstudio_model)
```

Comment:
I think this is ok for now but we may need to think of a different strategy for this section. Invariably, there will be some types of EnergyPlus/OpenStudio materials that are not supported in the Honeybee schema. Because Honeybee has default constructions that it resorts to when no construction has been specified, I think the best strategy will be to not assign any construction to a Face if it possesses a construction with an unsupported material. And we can just `puts` a warning that an object failed to translate.

Ultimately, we might be able to handle this all with `begin`/`rescue` loops when we go to make constructions from materials or apply constructions to Faces and Apertures. But it's just something to keep in mind since we might prefer to loop through all materials here at the start (no matter whether they are opaque, glazing, unsupported, etc). That way, we will know if there's an unsupported material in the model before we go to create a construction with it.

Reply:
Adding a TODO statement for now